### DISCUSSION TITLE
Welcome to GPT-3-Chatbot Discussions!

### DISCUSSION THREAD

**Discussion by pat8901** [pat8901/GPT-3-Chatbot#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ljairgt15** [comment]
Hi guys, I have a question, now I am in a project about chabot for the goverment, but the issue occurs is that when necessary, the bot will channel the conversation to a human agent, but not to any human agent, as it depends on some parameters that the user will send when filling out a survey about where he/she is located.

**Reply by pat8901** [reply]
Interesting, what exactly are you having trouble with? Creating prompts for GPT-3 can be tricky.
